src/journal-remote/microhttpd-util.h

```diff
@@ -41,6 +41,16 @@
 #  define MHD_USE_EPOLL MHD_USE_EPOLL_LINUX_ONLY
 #endif
 
+/* Renamed in µhttpd 0.9.52 */
+#ifndef MHD_USE_SSL
+#  define MHD_USE_TLS MHD_USE_SSL
+#endif
+
+/* Renamed in µhttpd 0.9.53 */
+#ifndef MHD_USE_POLL_INTERNALLY
+#  define MHD_USE_POLL_INTERNAL_THREAD MHD_USE_POLL_INTERNALLY
+#endif
+
```

Comment:
Sorry, I do not catch your comment.
The option `MHD_USE_POLL_INTERNALLY` is defined in very old date, and recently renamed to `MHD_USE_POLL_INTERNAL_THREAD`. These `#ifndef` lines intend to support old version of libmicrohttpd. If you use the latest library, then these lines do almost nothing.
https://gnunet.org/git/libmicrohttpd.git/tree/src/include/microhttpd.h#n985